### PR TITLE
RWA-585 Update task configuration to include case management category

### PR DESCRIPTION
**Pull request by mkayad**
### JIRA link (if applicable) ###

[RWA-585](https://tools.hmcts.net/jira/browse/RWA-585)

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

**Comment by iguannago** [comment]
there is no test for this change? Do we need to add any?

**Review by iguannago — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/wataskconfigurationapi/domain/entities/camunda/enums/CamundaVariableDefinition.java [hunk 1]
```diff
@@ -21,7 +21,8 @@
     FORM_KEY("formKey"),
     CASE_ID("caseId"),
     NAME("name"),
-    HAS_WARNINGS("hasWarnings");
+    HAS_WARNINGS("hasWarnings"),
+    CASE_MANAGEMENT_CATEGORY("caseManagementCategory");
```
[the remark this answers is not in the document]
Comment (mkayad) [reply src/main/java/uk/gov/hmcts/reform/wataskconfigurationapi/domain/entities/camunda/enums/CamundaVariableDefinition.java:25]: I added some more code

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/controllers/PostTaskConfigurationTest.java [hunk 2]
```diff
@@ -36,7 +36,7 @@ public void cleanUp() {
 
     @Test
     public void should_return_task_configuration_then_expect_task_is_auto_assigned() throws Exception {
-        caseId = createCcdCase();
+        caseId = createCcdCase("case_data.json");
```
Comment (alexbaeza) [inline src/functionalTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/controllers/PostTaskConfigurationTest.java:39]: Looks good minor suggestion to have an alternative to having to pass in the name of the json file, we could wrap this into:

createCcdCase()
createCcdCaseWithCaseManagementCategory()

What do you think?

**Review by alexbaeza — APPROVED** [review]

**Review by alexbaeza — DISMISSED** [review]

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/controllers/PostTaskConfigurationTest.java (hunk 2 above)
Comment (mkayad) [reply src/functionalTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/controllers/PostTaskConfigurationTest.java:39]: that is a good idea

**Review by indvamshi — APPROVED** [review]

**Pull request merged**